cmd/analyze.go

```diff
@@ -0,0 +1,80 @@
+// Copyright 2019 The OPA Authors.  All rights reserved.
+// Use of this source code is governed by an Apache2
+// license that can be found in the LICENSE file.
+
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+
+	"github.com/open-policy-agent/opa/ast"
+	"github.com/open-policy-agent/opa/internal/cmd/analyze"
+	pr "github.com/open-policy-agent/opa/internal/presentation"
+	"github.com/open-policy-agent/opa/loader"
+)
+
+var analyzeCommand = &cobra.Command{
+	Use:    "analyze <path>",
+	Short:  "Analyze Rego source code",
+	Hidden: true,
+
+	PreRunE: func(Cmd *cobra.Command, args []string) error {
+		if len(args) == 0 {
+			return fmt.Errorf("specify at least one file")
+		}
+		return nil
+	},
+
+	Run: func(cmd *cobra.Command, args []string) {
+		os.Exit(analyzeModules(args))
+	},
+}
+
+func analyzeModules(args []string) int {
+	modules := map[string]*ast.Module{}
+
+	f := loaderFilter{}
+
+	result, err := loader.Filtered(args, f.Apply)
```

Comment:
We should add bundle support, we started pushing all the CLI's to support `-b/--bundle` for loading modules so we should keep it up for new ones. It should be pretty easy to copy what we do in some of the other CLI's.